src/Process.js

```diff
@@ -77,10 +81,12 @@ Process.prototype.stream = T.func([], TDuplexStream, "process.stream").of(functi
 /**
  * Kills a running Process
  *
- * @param {String} signal to kill the process
+ * @param {MaybeExitCodes} maybeExitCodes that are valid
  */
-Process.prototype.kill = T.func([], T.Object, "process.kill").of(function() {
-    return this.instance.kill();
+Process.prototype.kill = T.func([MaybeExitCodes], T.Object, "process.kill").of(function(maybeExitCodes) {
```

Comment:
falls das doch klappt, überprüf mal, ob die prozesse auch wirklich beendet sind, weil falls hier auto-currying zieht, kanns auch sein, dass du, weil parameter fehlen ne funktion zurück kriegs, wodurch das await in den tests sofort resolved und dann done() gecalled wird